### ISSUE TITLE
Sagemaker warm start

### ISSUE THREAD

**Issue by ghost** [aws/amazon-sagemaker-examples#861]
I am tuning a neural net for a DeepAR based model. I have 3 tunable hyper-parameters, 10 static ones and 1 objective metric for the parent job as follows.

tuning_objective_metric	FreeText	-	test:RMSE
Cardinality	FreeText	-	auto
context_length	FreeText	-	30
dropout_rate	Continuous	Linear	0.0001 - 0.1
epochs	Integer	Linear	10 - 20
learning_rate	Continuous	Logarithmic	0.0001 - 0.001
likelihood	FreeText	-	deterministic-l1
mini_batch_size	Integer	Auto	32 -64
num_cells	Integer	Auto	40 - 70
num_dynamic_feat	FreeText	-	auto
num_eval_samples	FreeText	-	100
num_layers	FreeText	-	2
prediction_length	FreeText	-	30
test_quantiles	FreeText	-	http://0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9
time_freq	FreeText	-	1D
When I try to use warm start (without changing anything but epoch)and change the number of epochs to 10-40, I get following error:

The set of tunable plus static HyperParameter names is not the same across current and parent tuning jobs. Please ensure this set is constant when warm starting. Note that previously static HyperParameters can be tuned in warm start jobs and vice versa.

1) Isn't my set of tunable plus static HyperParameter names still the same with just change in search space of epochs? Do I not have 3 tunable hyper-parameters, 10 static ones and 1 objective metric?
2) Can I change the scale of drop out rate from linear to auto during warm start? 
3) Is it possible to keep everything as is except for number of epochs and change one static parameter to tunable, let's say num_layers in above example?

Amazon documentation says 
The number of total changes from tunable hyperparameters in the parent jobs to static hyperparameters in the new tuning job, plus the number of changes in the values of static hyperparameters cannot be more than 10.

Can you give an example of a scenario that I can change ? I understand case of categorical variables , not sure how it's counted for continuous/int?

**Comment by seanpmorgan** [comment]
Closing due to the creators account being closed so we can not discuss. Please feel to re-open if there is still an issue.
